client/my-sites/plans/jetpack-plans/jetpack-complete-page/components/cta-buttons/style.scss

```diff
@@ -0,0 +1,40 @@
+.jetpack-complete-page__cta-buttons {
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	width: 100%;
+	& > * {
+		flex: 1;
+	}
+	@include breakpoint-deprecated( "<960px" ) {
+		flex-direction: column;
+	}
+
+	.jetpack-complete-page__get-complete-button {
+		@include breakpoint-deprecated( ">960px" ) {
+			margin-right: 2em;
+		}
+		&.form-button {
+			color: var(--studio-white);
+			background-color: var(--studio-black);
+			&:hover,
+			&:active {
+				background-color: var(--studio-gray-90);
+			}
```

Comment:
I'm not sure about this background color mixed with the white text, but I don't see design specs for what it should be. @Automattic/jetpack-design, input here? (Ignore all the other aspects, those are addressed in other comments.)

<img width="296" alt="image" src="https://user-images.githubusercontent.com/670067/220655110-81b48de0-b4e7-494c-b976-4f23da48719b.png">